packages/calcite-components/src/components/switch/switch.scss

```diff
@@ -69,59 +70,48 @@
     relative
     box-border
     inline-block
-    border
-    border-solid
     align-top
     focus-base;
 
   border-radius: var(--calcite-switch-corner-radius, 9999px);
-  border-color: var(--calcite-switch-border-color, var(--calcite-color-border-2));
-  background-color: var(--calcite-switch-background-color, var(--calcite-color-foreground-2));
+  border-color: var(--calcite-switch-border-color);
+  background-color: var(--calcite-switch-background-color, var(--calcite-color-border-input));
 }
 
 .container:focus .track {
   @apply focus-outset;
 }
 
+.container:hover .track {
+  background-color: var(--calcite-switch-background-color-hover, var(--calcite-color-text-3));
+}
+
 .handle {
   @apply pointer-events-none
     absolute
     block
-    border-2
-    border-solid
     transition-all
     duration-150
     ease-in-out;
-  inset-block-start: -1px;
-  inset-inline: -1px theme("inset.auto");
+  inset-block-start: 2px;
+  inset-inline: 2px theme("inset.auto");
   background-color: var(--calcite-switch-handle-background-color, var(--calcite-color-foreground-1));
-  border-color: var(--calcite-switch-handle-border-color, var(--calcite-color-border-input));
+  border-color: var(--calcite-switch-handle-border-color);
   border-radius: var(--calcite-switch-corner-radius, 9999px);
   box-shadow: var(--calcite-switch-handle-shadow);
 }
 
-:host(:hover:not([disabled])),
-:host(:focus:not([disabled])) {
-  .handle {
-    border-color: var(--calcite-switch-handle-border-color, var(--calcite-color-brand));
-    box-shadow: var(--calcite-switch-handle-shadow, inset 0 0 0 1px var(--calcite-color-brand));
-  }
-}
-
 :host([checked]) {
   .track {
-    border-color: var(--calcite-switch-border-color, var(--calcite-color-brand-hover));
     background-color: var(--calcite-switch-background-color, var(--calcite-color-brand));
   }
   .handle {
-    border-color: var(--calcite-switch-handle-border-color, var(--calcite-color-brand));
-    inset-inline: theme("inset.auto") -1px;
+    inset-inline: theme("inset.auto") 2px;
   }
 
   &:host(:hover:not([disabled])) {
-    .handle {
-      border-color: var(--calcite-switch-handle-border-color, var(--calcite-color-brand-hover));
-      box-shadow: var(--calcite-switch-handle-shadow, inset 0 0 0 1px var(--calcite-color-brand-hover));
```

Comment:
I think the var `--calcite-switch-handle-shadow` still needs to be honored here even though its deprecated. right? I was just referring to removing the default value.

Reply:
nevermind. ignore this comment :)